lib/code_corps/donation_goals_manager.ex

```diff
@@ -0,0 +1,114 @@
+defmodule CodeCorps.DonationGoalsManager do
+  import Ecto.Query
+
+  alias CodeCorps.{DonationGoal, Project, Repo}
+  alias Ecto.Multi
+
+  def create(attributes) do
+    changeset = %DonationGoal{} |> DonationGoal.create_changeset(attributes)
+
+    multi = Multi.new
+    |> Multi.insert(:donation_goal, changeset)
+    |> Multi.run(:update_current_goal, &transactional_update_current_goal/1)
+
+    case Repo.transaction(multi) do
+      {:ok, %{donation_goal: donation_goal, update_current_goal: _}} ->
+        {:ok, Repo.get(DonationGoal, donation_goal.id)}
```

Comment:
Was asking Josh about this since I wasn't sure if I missed something. `Multi.insert` should be returning a `%DonationGoal{}` under the `:donation_goal` key.